src/test/run-pass/issue-8460.rs

```diff
@@ -24,21 +35,21 @@ macro_rules! check {
 
 fn main() {
     check![
-        isize::min_value() / -1,
-        i8::min_value() / -1,
-        i16::min_value() / -1,
-        i32::min_value() / -1,
-        i64::min_value() / -1,
+        isize::min_value() / -isize::one(),
+        i8::min_value() / -i8::one(),
+        i16::min_value() / -i16::one(),
+        i32::min_value() / -i32::one(),
+        i64::min_value() / -i64::one(),
```

Comment:
Ah. Because I needed a local `zero` implemention anyway, so adding `one` is easy and gets rid of the compile-time warning that this is going to overflow.  We could also go the other direction and use plain literals for all, which will then warn on every line.  Or we could leave it as-is with `zero()` and `-1`, but the inconsistency bothered me. 😀

I don't think it matters to the actual test.